sparkbox/designsystemssurvey
# Enforce SSL

## Steps to Test

1.  Run the server as usual (`npm run server`, etc)
2. Navigate to http://localhost:3000
**Expected:** You see the site without SSL

1. Run the server with the `ENFORCE_SSL` environment variable (`ENFORCE_SSL=true npm run server`)
2. Navigate to http://localhost:3000
**Expected:** Redirected to https://localhost:3000 
  ⚠️ Site will not appear since we are not serving a certificate in local dev

## Production deployment/settings
In Heroku, we will need to create the `ENFORCE_SSL` setting. This has already been done and should Just Work™
